shared/clinical-code-sets/medications/antipsychotics/2/antipsychotics.ctv3.txt

```diff
@@ -0,0 +1,685 @@
+d915.	Parstelin tablet
+d91D.	Amitriptyline hydrochloride+perphenazine 10mg/2mg tablet
+d917.	Triptafen-M tablet
+d91C.	Amitriptyline hydrochloride+perphenazine 25mg/2mg tablet
+d916.	Triptafen tablet
+d91A.	Fluphenazine hydrochloride+nortriptyline 500micrograms/10mg tablet
+d914.	Motival tablet
+d919.	Fluphenazine hydrochloride+nortriptyline 1.5mg/30mg tablet
+x00fc	Motipress tablet
+d913.	Motipress tablets X28CP
+d58..	OLANZAPINE PAMOATE
+d583.	ZYPADHERA 405mg powder+solvent for suspension for injection
+d582.	ZYPADHERA 300mg powder+solvent for suspension for injection
+d581.	ZYPADHERA 210mg powder+solvent for suspension for injection
+d58z.	OLANZAPINE 210mg powder+solvent for suspension for injection
+d58y.	OLANZAPINE 300mg powder+solvent for suspension for injection
+d58x.	OLANZAPINE 405mg powder+solvent for suspension for injection
+x03wV	Clozaril
+x02Xe	Clopixol Conc
+x02Xc	Clopixol
+x02Xd	Clopixol Acuphase
+x048H	Stemetil Eff
+x02rY	Stemetil
+x02rK	Sparine
+x02rX	Stelazine
+x05up	Denzapine
+x02Z4	Depixol Low Volume
+x02Z3	Depixol Conc
+x02Z2	Depixol
+x004Z	Fentanyl+droperidol
+o42w.	Fentanyl+droperidol 100micrograms/5mg/2mL injection
+o423.	Thalamonal injection
+x056z	Zyprexa Velotab
+x03ik	Zyprexa
+x04wK	Zoleptil
+x048g	Sulpitil
+x02rk	Sulparex
+x05ZZ	Sulpor
+x02aZ	Droleptan
+x02aV	Dozic
+x02aH	Dolmatil
+x02ps	Risperdal
+x05gE	Risperdal Consta
+x05kh	Risperdal Quicklet
+x048x	Haldol Concentrate
+x02eN	Haldol decanoate
+x02eM	Haldol
+x02je	Modecate Concentrate
+x02jd	Modecate
+x02jf	Moditen
+h8A..	NATALIZUMAB
+h8Az.	NATALIZUMAB 300mg/15mL solution for injection
+h8A1.	TYSABRI 300mg/15mL solution for injection
+h8P..	VEDOLIZUMAB
+h8P1.	ENTYVIO 300mg powder for concentrate for soln for infusion
+h8P2.	VEDOLIZUMAB 300mg pdr for concentrate for soln for injection
+x02qq	Serenace
+x03hI	Serdolect
+x02kn	Neulactil Forte
+x02km	Neulactil
+x02h3	Largactil Forte
+x02h2	Largactil
+x05Zv	Buccastem M
+x02WA	Buccastem
+x02jy	Motival
+x03sw	Motipress
+x02tP	Triptafen-M
+x02tO	Triptafen
+x05Ku	Amisulpride - chemical
+x05Lt	Benperidol - chemical
+X77Wv	Chlorpromazine stimulation test
+dj24.	Diamorphine/cocaine/chlorpromazine elixir
+dj26.	Morphine/cocaine/chlorpromazine elixir
+XaJPR	Clozapine monitoring
+gd6..	Nortriptyline hydrochloride [enuresis - no drugs here]
+XaE6Z	Thioridazine retinopathy
+x02iZ	Melleril
+x03qI	Seroquel
+x02m5	Orap
+x02cS	Fentazin
+X80fX	Integrin beta 1 subunit
+x02hz	Loxapac
+x03dK	Kentace
+x05w0	Zaponex
+x05te	Abilify
+x02nW	Piportil Depot
+x02p5	Redeptin
+x02XC	Chloractil
+x05h1	Benquil
+x02U0	Anquil
+x03rg	Solian
+x02oi	Proziere
+dhg2.	VERTIGON 25mg tablets
+x02cz	Fluanxol
+x02pc	Rideril
+x02mx	Parstelin
+d84z.	Tranylcypromine 10mg tablet
+d84z.	Tranylcypromine 10mg tablet
+dw19.	Tranquilyn 5mg tablet
+dw19.	Tranquilyn 5mg tablet
+dw1A.	Tranquilyn 10mg tablet
+dw1A.	Tranquilyn 10mg tablet
+dw1B.	Tranquilyn 20mg tablet
+dw1B.	Tranquilyn 20mg tablet
+X80fL	Beta chain of CD11 a, b and c
```

Comment:
X80fL and X80gC seem unrelated to this code set - please remove all instances of them

Reply:
Also in the EMIS and the SNOMED sets remove the ones referring to "beta chain" and "antigen"